problem-solver/cxx/exampleModule/specifications/agent_of_isomorphic_search/ui_menu_isomorphic_search.scsi

```diff
@@ -0,0 +1,29 @@
+ui_menu_isomorphic_search <- ui_user_command_class_atom; ui_user_command_class_view_kb; ui_one_argument_command_class;;
```

Comment:
Do you add this command to any menu in UI?

Reply:
I had been add this command in pattern UI.

Reply:
What do you mean by "pattern ui"?